### ISSUE TITLE
How to change language?

### ISSUE THREAD

**Issue by pullopen** [Plume-org/Plume#830]
**Is your feature request related to a problem? Please describe.**
I see translation files in the codes. But I can not find how to change the language in anywhere, neither `.env` nor web-ui. Is there a way to change the language?

**Describe the solution you'd like**
Maybe a way for administrator and user to change default and user's language?

**Additional context**
Version: 0.6.1-dev. Running on docker.

**Comment by KitaitiMakoto** [comment]
Currently, you need change your browser's language priority. Sorry for inconvenience.

#785 is related.

**Comment by pullopen** [comment]
> 
> 
> Currently, you need change your browser's language priority. Sorry for inconvenience.
> 
> #785 is related.

Sorry, tried on Firefox and Chrome when I set my browser's language to Chinese, still doesn't work... 

Curiously when I open your site, it is automatically set as Japanese...

**Comment by KitaitiMakoto** [comment]
Hmm... I will investigate the problem. Sorry for inconvenience.

**Comment by KitaitiMakoto** [comment]
We are so sorry. There are Chinese translations, but we don't enable them. I add them and push to Git repo soon.

**Comment by KitaitiMakoto** [comment]
I pushed some languages support to the main branch. You can check it out and build with Chinese support.

**Comment by pullopen** [comment]
> 
> 
> I pushed some languages support to the main branch. You can check it out and build with Chinese support.

Yes it's now solved. Thank you for your effort! I would try to contribute to some translation...

**Comment by pullopen** [comment]
> I pushed some languages support to the main branch. You can check it out and build with Chinese support.

Strangely though, I usually use Chinese-Simplified and it works well before v0.7.1, but when I upgrade it to v0.7.2, the interface automatically became Chinese-Traditional. It's not a big deal, but may I ask what happened?

**Comment by KitaitiMakoto** [comment]
Oh, it's not intended. We'are sorry. We will fix it.
